package.json

```diff
@@ -14,9 +14,7 @@
     "prerelease": "yarn build",
     "release": "changeset publish",
     "release:canary": "(node scripts/canary-release.js && yarn build && yarn changeset publish --tag alpha) || echo Skipping Canary...",
-    "zip": "yarn build && yarn --cwd packages/components pack",
-    "prepare": "npx husky install",
```

Comment:
Why the "husky install" was removed? it is needed for husky v6 to work

Reply:
Husky threw me an error, so I initially added `npx husky install`, but then it worked again without it. If it is mandatory, I can add it back. I didn't see it on main branch.